### PR TITLE
Fix log4j output nothing issue

### PR DESCRIPTION
**Pull request by maobaolong**
With this PR, we can specify log4j configure file by jvm argument `-Dlog4j.configuration=file:<LOG4j_FILE_PATH>`.

### CODE REVIEW

**Comment by maobaolong** [comment]
@Xushaohong @captainzmc  Please take a look at this PR, thanks.

File: conf/log4j.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 hcfsfuse.logger.type=TEST_LOGGER
-hcfsfuse.root.logger=INFO, ${hcfsfuse.logger.type}
+hcfsfuse.root.logger=${hcfsfuse.logger.type}
 
-log4j.rootLogger=${hcfsfuse.root.logger},stdout
+log4j.rootLogger=WARN, ${hcfsfuse.root.logger},CONSOLE
```
Comment (Xushaohong) [inline conf/log4j.properties]: Default to INFO might be better? Error>Warn>Info>Debug

**Review by Xushaohong — COMMENTED** [review]
LGTM +1

File: conf/log4j.properties (hunk 1 above)
Comment (maobaolong) [reply conf/log4j.properties]: @Xushaohong Thanks for this suggestion.

**Comment by maobaolong** [comment]
@Xushaohong The default log level has been changed from WARN to INFO, PTAL.

**Review by captainzmc — APPROVED** [review]
+1

**Review by Xushaohong — APPROVED** [review]
+1

**Pull request merged**